### PR TITLE
Small tweaks for WebDavClientService and tests

### PR DESCRIPTION
**Pull request by PVince81**
Follow up of @DeepDiver1975's review: https://github.com/owncloud/core/pull/29220#issuecomment-338615518

### CODE REVIEW

File: tests/lib/Files/Storage/DavTest.php [hunk 1]
```diff
@@ -50,37 +50,37 @@
...
 
 	/**
-	 * @var IClientService
+	 * @var IClientService | PHPUnit_Framework_MockObject_MockObject
 	 */
 	private $httpClientService;
 
 	/**
-	 * @var IWebdavClientService
+	 * @var IWebDavClientService | PHPUnit_Framework_MockObject_MockObject
```
Comment (DeepDiver1975) [inline tests/lib/Files/Storage/DavTest.php]: should be \PHPUnit_Framework_MockObject_MockObject
or add use PHPUnit_Framework_MockObject_MockObject at the beginning of the file

**Comment by PVince81** [comment]
@DeepDiver1975 adjusted, thanks

**Comment by PVince81** [comment]
@DeepDiver1975 I've fixed the conflicts, please recheck

**Comment by PVince81** [comment]
wat...

```
Test\Share20\ManagerTest::testGetAllSharesByExpiration
Failed asserting that actual size 201 matches expected size 200.

/var/lib/jenkins/workspace/owncloud-core_core_PR-29327-Q6FFR2VECSMTGU2DFJ4BZ4ZUAUNNMKTH2FE3RUZXHTZM4NN7CKWQ/tests/lib/Share20/ManagerTest.php:2031
```

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**